nomenclature/validation_schemas/variable_schema.yaml

```
@@ -17,11 +17,15 @@ definitions:
         # The lower-level dictionary are the attributes
         properties:
           unit:
-            type: [ string, "null" ]
+            oneOf:
+              - type: [ string, "null" ]
+              - type: array
+                items:
+                  type: string
         required:
           - unit
         additionalProperties:
-          type: [ string, number, "null" ]
+          type: [ string, number, boolean, "null" ]
```

Since you say in the description of the PR that a variable can also have a list as an attribute value, is here and `array` missing?
```suggestion
          type: [ array, string, number, boolean, "null" ]
```

---

No, only the unit can be a list at this point, all other attributes of a variable cannot be, because I encountered an issue with that - it's not needed right now, so let's keep that for another iteration.

---

Ah ok, then I would suggest editing the PR description because under the second point I read it as any variable attribute can be a list.

---

Done